axiom/nr_txn.c

```diff
@@ -531,6 +532,7 @@ nrtxn_t* nr_txn_begin(nrapp_t* app,
   nt->intrinsics = nro_new_hash();
 
   nt->custom_events = nr_analytics_events_create(app->limits.custom_events);
+  nt->log_events = nr_log_events_create(1000 /* app->limits.log_events */);
```

Comment:
Is this just a placeholder for a dependency on another PR?

Reply:
Correct. And actually since #497 went in, I might as well address it.